src/main/scala/algolia/objects/RenderingContent.scala

```diff
@@ -0,0 +1,88 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016 Algolia
+ * http://www.algolia.com/
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package algolia.objects
+
+/**
+  * Content defining how the search interface should be rendered. This is set via the settings for a default value
+  * and can be overridden via rules.
+  */
+case class RenderingContent(
+    facetOrdering: Option[FacetOrdering] = None
+)
+
+/**
+  * Facets and facets values ordering rules container.
+  */
+case class FacetOrdering(
+    facets: FacetsOrder,
+    values: Map[String, FacetValuesOrder]
+)
+
+/**
+  * Define or override the way facet attributes are displayed.
+  */
+case class FacetsOrder(
+    order: Seq[String]
+)
+
+/**
+  * Facet values ordering rule container.
+  */
+case class FacetValuesOrder(
+    order: Seq[String] = Seq.empty,
+    sortRemainingBy: SortRule = SortRule.count
```

Comment:
it doesn't default to count, it default to nothing, so this would be something like:

```suggestion
    sortRemainingBy: Option[SortRule]
```

Reply:
I've checked the spec, it says:
> * order is optional. Default: (empty)
> * sortRemainingBy is mandatory. Its Default: count

Maybe I am not looking at the right place?

Reply:
The default in the dashboard is count, but that shouldn't be the default when reading. That should be left over to the user IMO (same as the swift client where the default was moved to InstantSearch, cc @VladislavFitz)

Reply:
I've discussed this with @VladislavFitz, and fixed it :)